src/libraries/System.Security.Cryptography.X509Certificates/src/Resources/Strings.resx

```diff
@@ -295,6 +295,12 @@
   <data name="Cryptography_X509_PKCS7_Unsupported" xml:space="preserve">
     <value>PKCS#7 certificate format is not supported on this platform.</value>
   </data>
+  <data name="Cryptography_X509_PKCS12_PersistKeySetNotSupported" xml:space="preserve">
+    <value>PKCS#12 persist key set flag is not supported on this platform.</value>
```

Comment:
Personally, I'd use the enum member name as-is, which also solves a "keyset" vs "key set" spelling debate and a "(persist key set) flag" vs "(persist key) (set flag)" human parsing error.  I'd also add a "The" because... I don't know, it sounds better.

```suggestion
    <value>The PKCS#12 PersistKeySet flag is not supported on this platform.</value>
```

Similar with Exportable.

Reply:
That makes sense, changed!